docs/v6-glm-py/privacy.rst

```diff
@@ -0,0 +1,125 @@
+Privacy
+=======
+
+.. _privacy-guards:
+
+Guards
+------
+
+There are several guards in place to protect the privacy of individuals in the database:
+
+- **Minimum number of data rows to participate**: A node will only participate if it
+  contains at least `n` data rows. This is to prevent nodes with very little data from
+  participating in the computation. By default, the minimum number of data rows is set
+  to 10. Node administrators can change this minimum by adding the following to their
+  node configuration file:
+
+  .. code:: yaml
+
+    algorithm_env:
+      GLM_MINIMUM_ROWS: 10
+
+  Note that the algorithm also checks for each of the columns involved in generating the
+  model that the column contains at least this number of non-null values. If not, the
+  computation is also refused.
+
+- **Minimum number of organizations to participate**: The minimum number of
+  organizations to participate in a GLM computation is set to 3. This is to prevent
+  that a single organization can try to infer the data of only one other organization
+  involved in the computation. Node administrators can change this minimum by adding the
+  following to their node configuration file:
+
+  .. code:: yaml
+
+    algorithm_env:
+      GLM_MINIMUM_ORGANIZATIONS: 3
+
+- **Check parameters vs observations ratio**: If the number of parameters is high
+  compared to the number of observations, the computation will not be allowed. This is
+  to prevent that data may be inferred from an overfitted model. The maximum ratio of
+  parameters vs observations is set to 10%. Node administrators can change this ratio
+  using the following configuration:
+
+  .. code:: yaml
+
+    algorithm_env:
+      ENVVAR_MAX_PCT_PARAMS_OVER_OBS: 10
```

Comment:
Why the weird `ENVVAR_` prefix instead of the `GLM_`?

Reply:
Oops should be `GLM_MAX_PCT_VARS_VS_OBS` - I copied the variable name of the constant rather than its value